util/chunk/chunk.go

```diff
@@ -394,7 +371,8 @@ func (c *Chunk) AppendRowByColIdxs(row Row, colIdxs []int) (wide int) {
 	return
 }
 
-// AppendPartialRowByColIdxs appends a row by its colIdxs to the chunk.
+// AppendPartialRowByColIdxs appends a row to the chunk starting from colOff,
+// using the row‘s columns specified by colIdxs.
```

Comment:
looks like it is a Chinese single quote 😄
```suggestion
// using the row's columns specified by colIdxs.
```